src/docs/product/issues/performance-issues/index.mdx

```diff
@@ -1,22 +1,22 @@
 ---
-title: "Performance Issues"
+title: "Performance Issue Details"
 sidebar_order: 15
-description: "Learn more about Performance Issues and how to use them."
+description: "Learn more about what information displays on the Issue Details page to help you efficiently triage a performance issue."
 ---
 
-<Include name="early-adopter-note.mdx" />
-
-The **Issue Details** page for performance issues helps you to gain further insight into the source of an issue and the impact it has on your application's users. The main area of the page displays information about a specific transaction event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the events grouped together in this issue:
+The **Issue Details** page for performance issues helps you to gain further insight into the source of an issue and the impact it has on your application's users. The main area of the page displays information about a specific transaction event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the transaction events grouped together in this issue:
 
 ![Performance issue](performance-issue-details.png)
 
+<!-- update image for GA if necessary -->
```

Comment:
@gggritso - replace image and remove or just remove if image doesn't need updating.

Reply:
Done